packages/yoastseo/src/languageProcessing/helpers/highlighting/getMarkingsInSentence.js

```diff
@@ -0,0 +1,154 @@
+import Mark from "../../../../src/values/Mark";
```

Comment:
Maybe now we can visually indicate the separation through a comment divider for now (examples [here](https://github.com/stackbreak/comment-divider)), because indeed, as some point we will deprecate / remove search-based highlighting.

Reply:
I don't know if we can visually separate them now. They are entangled. This is because when we output the position-based highlighting, now we also still make it compatible with search-based highlighting. This is to make sure that in other editors that we still haven't adapted, we can still use the search-based highlighting.
The most that I can do now is providing clearer comment